Library/Formula/textql.rb

```diff
@@ -0,0 +1,32 @@
+require "language/go"
+
+class Textql < Formula
+  desc "Executes SQL across text files"
+  homepage "https://github.com/dinedal/textql"
+  url "https://github.com/dinedal/textql/archive/2.0.0.tar.gz"
+  sha256 "c9ea3c35332163be54d3b742ee8a691e82f10dbc0a4f3a76c35c1480787cd952"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/mattn/go-sqlite3" do
+    url "https://github.com/mattn/go-sqlite3.git",
+      :revision => "8897bf145272af4dd0305518cfb725a5b6d0541c"
+  end
+
+  def install
+    (buildpath + "src/github.com/dinedal/textql").install "inputs", "outputs", "storage", "sqlparser", "util", "cmd"
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-i", "-ldflags", "-X main.VERSION 2.0.0", "#{buildpath}/src/github.com/dinedal/textql/cmd/textql.go"
+    bin.install "textql"
+  end
+
+  test do
+    version = pipe_output("#{bin}/textql --version")
+    assert_match /2.0.0/, version
+    word_count_wc = pipe_output("wc -l /usr/share/dict/words | cut -d\" \" -f 3")
+    word_count_textql = pipe_output("#{bin}/textql -sql 'select count(*) from words' /usr/share/dict/words")
```

Comment:
Please create a dummy file here so we don’t rely on `/usr/share/dict/words` being present. Something like the following should work:

```rb
(testpath/"test").write "a\nb\nc\n"
assert_match "3\n", shell_output("#{bin}/textql -sql 'select count(*) from test' #{testpath}/test")
```